libraries/chain/hotstuff/finalizer.cpp

```diff
@@ -58,29 +60,29 @@ finalizer::vote_decision finalizer::decide_vote(const block_state_ptr& proposal,
       safety_check   = false;
    }
 
-   dlog("liveness_check=${l}, safety_check=${s}, monotony_check=${m}",
-        ("l",liveness_check)("s",safety_check)("m",monotony_check));
+   dlog("liveness_check=${l}, safety_check=${s}, monotony_check=${m}, ${can} vote",
+        ("l",liveness_check)("s",safety_check)("m",monotony_check)("can",(liveness_check || safety_check)?"can":"cannot"));
```

Comment:
I actually prefer `vote=` instead of the "fancy", `can/cannot`. Easier to see in the log output.

Reply:
At this point I don't know what the vote will be, just that we can or cannot vote.
If we can vote, the vote is spelled out [below](https://github.com/AntelopeIO/leap/blob/a512f1793e9bcc48ecf4407ca5e2675cd5128228/libraries/chain/hotstuff/finalizer.cpp#L94-L95)

Reply:
I just meant `vote=true`, `vote=false`

Reply:
I actually think this is less clear. What about `can vote = true`, `can vote = false`?

Reply:
Yes, that is fine, or even just leave it how you had it, not a big deal.